File: Web/RestaurantOnline/Model.RestaurantOnline/CategoryModel.cs
using System;
namespace Model.RestaurantOnline
{
    public class CategoryModel
    {
        private int id;
        private string eName;
        private string otherName;

        public int Id{
            get
            {
                return id;
            }

            set
            {
                this.id = value;
            }
        }

        public string EName{
            get
            {
                return eName;
            }

            set
            {
                this.eName = value;
            }
        }

        public string OtherName{
            get
            {
                return otherName;
            }

            set
            {
                this.otherName = value;
            }
        }
    }
}

File: Android.RestaurantMenu/app/src/main/java/com/bojinzhang/android/Model/CategoryModel.java
package com.bojinzhang.android.Model;

/**
 * Created by zhangbojin on 25/05/17.
 */

public class CategoryModel {
    private String eName;
    private String oName;
    private int id;

    public String getEName() {
        return eName;
    }

    public String getOName() {
        return oName;
    }

    public int getId() {
        return id;
    }


    public CategoryModel(String eName, String oName, int id) {
        this.eName = eName;
        this.oName = oName;
        this.id = id;
    }
}
